client/internal/storage/repository/init.go

```diff
@@ -0,0 +1,23 @@
+package repository
+
+import (
+	"client/internal/logger"
+	"database/sql"
+
+	_ "github.com/lib/pq"
+)
+
+type Storage struct {
+	driver *sql.DB
+}
+
+func Make(dsn string) (Storage, error) {
```

Comment:
Я бы рекомнедловал тут вернуть * хотя бы потому, что работа идет везде с указателем.
```suggestion
func Make(dsn string) (*Storage, error) {
```

Reply:
Согласен, исправил